tensorboard/webapp/plugins/text/text_v2_plugin.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Sorry for not catching this earlier. Could you move the plugin python files to `tensorboard/plugins`? So it should end up in `tensorboard/plugins/text_v2`. The rationale is, sometimes, the plugins contain Python API that people use from the TensorFlow side. As such, it is likely that we will never move the plugin Python files out of the `tensorboard/plugins` while FE code have a bit more leeway. 

I think we are at a point where the benefit colocating FE code is stronger so we want the Angular frontend code to be in `tensorboard/webapp` but the Python counterpart should stay in the `tensorboard/plugins`.

Reply:
done